src/ComputeSharp.D2D1/Shaders/Interop/D2D1ShaderCompilerOptions.cs

```diff
@@ -0,0 +1,172 @@
+﻿using System;
+using TerraFX.Interop.DirectX;
+
+namespace ComputeSharp.D2D1.Interop;
+
+/// <summary>
+/// Flags to control the options that can be used to compile a D2D1 shader.
```

Comment:
I'd also put a note in here describing what options are used by CS.D2D1 when compiling `struct` shaders, so folks have a starting point to work from

Reply:
Oh, I see you already have a `Default` value :)

Reply:
Yup! I've also added remarks to clarify when to use or not use `PackMatrixColumnMajor`.